CODEOWNERS

```
@@ -0,0 +1 @@
+
```

```suggestion
#* @hashicorp/team-self-managed-release-engineering
```

TODO: Get a team in GH!